spec/components/utility/gcse_qualification_cards_component_spec.rb

```diff
@@ -47,6 +47,28 @@
       end
     end
 
+    context 'when failed required gcse' do
```

Comment:
Small tweak just to make clear that "D" isn't a "fail", but just below the ITT standard.

```suggestion
    context 'when their GCSE grade is below C or 4' do
```